README.rst

```diff
@@ -5,23 +5,21 @@ Ansible
 *******
 
 Ansible is a radically simple IT automation system. It handles
-configuration-management, application deployment, cloud provisioning,
-ad-hoc task-execution, and multinode orchestration -- including
-trivializing things like zero-downtime rolling updates with load
-balancers.
+configuration management, application deployment, cloud provisioning,
+ad-hoc task execution, and multi-node orchestration. Ansible makes complex
+changes like zero-downtime rolling updates with load balancers easy. More information on `the Ansible website <https://ansible.com/>`_.
 
-Read the documentation and more at https://ansible.com/
+You can install a released version of Ansible via ``pip``, a package manager, or
+our `release repository <https://releases.ansible.com/ansible/>`_. See our
+`installation guide <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>`_ for details on installing Ansible
+on a variety of platforms.
 
-You can find installation instructions
-`here <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>`_ for a
-variety of platforms.
+Red Hat offers supported builds of `Ansible Engine <https://www.ansible.com/ansible-engine>`_.
 
-Most users should probably install a released version of Ansible from ``pip``, a package manager or
-our `release repository <https://releases.ansible.com/ansible/>`_. `Officially supported
-<https://www.ansible.com/ansible-engine>`_ builds of Ansible are also available. Some power users
-run directly from the development branch - while significant efforts are made to ensure that
-``devel`` is reasonably stable, you're more likely to encounter breaking changes when running
-Ansible this way.
+Power users and developers can run the development
+branch directly. It is reasonably stable, but you are more likely to encounter
+breaking changes when running the ``devel`` branch. We recommend participating
+in the Ansible community if you want to run the development branch - see Get Involved below.
```

Comment:
```suggestion
in the Ansible community if you want to run the development branch — see Get Involved below.
```

Reply:
"Get Involved" could be referenced using a relative anchor

Reply:
Do relative anchors show up on GitHub?

Reply:
@acozine I believe so

Reply:
I changed the order of the sections to address this concern without an internal link. See 1f59d0c0118c11a56cb1a4d760b176290287e12e.